src/homepageExperience/components/steps/go/ExecuteAggregateQuery.tsx

```diff
@@ -24,7 +24,7 @@ export const ExecuteAggregateQuery = (props: OwnProps) => {
   |> mean()`
 
   const query = `
-query := \`from(bucket: "${bucket}")
+query = \`from(bucket: "${bucket}")
```

Comment:
`query` was already defined so we don't need to recreate it using `:=`. Just a reassignment is fine